logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java

```diff
@@ -161,4 +161,12 @@ public void eth1ServiceDown(final long interval) {
   public void eth1AtHead() {
     log.info("Eth1 tracker successfully caught up to chain head");
   }
+
+  public void usingGeneratedP2pPrivateKey(final String keyKey, final boolean justGenerated) {
```

Comment:
nit: `keyKey`? Maybe just `key`?

Reply:
It's a key for a private key 😄 
But ok, will change